R/lazyframe-frame.R

```diff
@@ -264,14 +267,14 @@ lazyframe__sort <- function(
   })
 }
 
-#' Add columns to this LazyFrame
+#' Modify/append column(s) of a LazyFrame
 #'
-#' Added columns will replace existing columns with the same name.
+#' Add columns or modify existing ones with expressions. This is
+#' the equivalent of `dplyr::mutate()` as it keeps unmentioned columns (unlike
+#' `$select()`).
```

Comment:
One major difference from `dplyr::mutate` is that in `mutate` you can reference added columns within the same function, which is not possible here.